packages/amplify_datastore/example/ios/unit_tests/resources/modelSchema/model_schema_maps.json

```diff
@@ -1,279 +1,304 @@
-                "type": {
-                    "fieldType": "string"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "owner": {
-                "name": "owner",
-                "type": {
-                    "fieldType": "string"
-                },
-                "isRequired": false,
-                "isArray": false
-            }
-        }
-    },
-
-    "AllTypeModelSchema": {
-        "name": "AllTypeModel",
-        "pluralName": "AllTypeModels",
-        "fields": {
-            "id": {
-                "name": "id",
-                "type": {
-                    "fieldType": "string"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "stringType": {
-                "name": "stringType",
-                "type": {
-                    "fieldType": "string"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "intType": {
-                "name": "intType",
-                "type": {
-                    "fieldType": "int"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "floatType": {
-                "name": "floatType",
-                "type": {
-                    "fieldType": "double"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "boolType": {
-                "name": "boolType",
-                "type": {
-                    "fieldType": "bool"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "dateType": {
-                "name": "dateType",
-                "type": {
-                    "fieldType": "date"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "dateTimeType": {
-                "name": "dateTimeType",
-                "type": {
-                    "fieldType": "dateTime"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "timeType": {
-                "name": "timeType",
-                "type": {
-                    "fieldType": "time"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "timestampType": {
-                "name": "timestampType",
-                "type": {
-                    "fieldType": "timestamp"
-                },
-                "isRequired": true,
-                "isArray": false
-            },
-            "enumType": {
-                "name": "enumType",
-                "type": {
-                    "fieldType": "string"
-                },
-                "isRequired": true,
-                "isArray": false
-            }
-        }
-    }
-}
+{

```

Comment:
I think the IDE accidentally converted this file into CRLF mode. (most files are in LF mode in the codebase). Could you double check please?

Reply:
Hey this is quite strange.  I did not find a reason for why this happened. 

Do you know how I might verify if it's in CRLF mode?  I think that if my computer / IDE was making those changes that there might be other issues in all of my other commits / files changed though. 

Looking through the file history you'll see other commits have had the same issue as well: 

![image](https://user-images.githubusercontent.com/11478656/129422731-bb4999ac-a74c-426f-8535-e35afca115ff.png)

Reply:
The json file was in CRLF mode, and was converted into LF mode to align with codebase in commit `ef250295`.
Could you check you git config if it uses CRLF mode? And the IDE you are using (e.g. prettier in VSCode has option to reformat with LF vs CRLF)

Reply:
Alright where in git config do I check for this CRLF mode.  I don't see any features in the config file that would be enabling that.  I feel like if that was the case I would have had issues in all my other commits though right? 

I installed prettier in VSCode, the default is set to LF mode.  I did auto format on save and switched between CRLF and LF modes but git did not register any file changes so I'm a bit unsure what's going on there.  

How would you switch the file to LF mode?